app/migrations/20250306122357-adds_organization_table.cjs

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("Organization", {
+      organization_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      name: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      contact_email: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      contact_number: {
+        type: Sequelize.TEXT,
+        allowNull: true,
+      },
```

Comment:
Let's make sure we're not exposing this field publicly. I think Sequelize has hooks that you can overwrite in the Model to make sure certain keys are removed before models are serialized.